powershell/VstsTaskSdk/InputFunctions.ps1

```diff
@@ -67,7 +69,7 @@ function Get-Input {
         [switch]$AsInt)
 
     $PSBoundParameters['Name'] = (Get-LocString -Key PSLIB_Input0 -ArgumentList $Name)
-    Get-EnvVariable @PSBoundParameters -Path "Env:INPUT_$($Name.Replace(' ', '_').ToUpperInvariant())"
+    Get-SecureInput @PSBoundParameters -Path "Env:INPUT_$($Name.Replace(' ', '_').ToUpperInvariant())"
```

Comment:
The func at the bottom is common code shared between Get-TaskVariable and Set-TaskVariable. For some reason the other functions (Get-Input/Get-Endpoint) have different formatting rules.